src/main/java/core/KitchenSimulator.java

```diff
@@ -0,0 +1,147 @@
+package core;
+
+import controllers.InventoryItemController;
+import controllers.OrderController;
+import controllers.WaiterController;
+import entities.*;
+import managers.OrderQueueManager;
+import models.StatusOrder;
+import utils.IoC;
+import utils.Utils;
+
+import java.util.List;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+public class KitchenSimulator {
+
+    private static IoC ioc = IoC.getInstance();
+    private static OrderQueueManager orderQueueManager = new OrderQueueManager();
+
+    public static void main(String[] args) throws InterruptedException {
+        Utils.init();
+
+        KitchenSimulator kitchenSimulator = new KitchenSimulator();
+
+        while (true) {
+            kitchenSimulator.moveOrdersToPendingQueue();
+            kitchenSimulator.executeWaiterJob();
+
+            Thread.sleep(5000);
+        }
+
+    }
+
+    public void moveOrdersToPendingQueue() {
+        var orderController = ioc.<OrderController>getInstance("OrderController");
+
+        List<Order> orders = orderController.findOrdersByStatus(null);
```

Comment:
Do you really need to specify `null`? I think you can infer null if no args are specified.